perftests/network-ssl/Makefile.BSD

```diff
@@ -46,6 +46,11 @@ IDIRS	+=	-I ${LIBCPERCIVA_DIR}/network
 SRCS	+=	network_ssl.c
 IDIRS	+=	-I ${LIB_DIR}/network_ssl
 
+# SSL compatibility
```

Comment:
I would move this up one block.  I have a very rough practice of ordering layers from low level to higher level.

Reply:
done